### PR TITLE
Presented Selenium as one possible WebDriver testing framework

### PR DESCRIPTION
**Pull request by mikehoffms**

### CODE REVIEW

File: microsoft-edge/webdriver-chromium/index.md [hunk 1]
```diff
@@ -65,17 +65,22 @@ To begin automating tests, use the following steps to ensure that the WebDriver
...
+
+The last component you must download is a language-specific client driver to translate your code \(Python, Java, C\#, Ruby, JavaScript\) into commands that the Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  
 
-[Download the WebDriver language binding of your choice][SeleniumDownloads].  The Microsoft Edge team recommends [Selenium 4.0.0-beta2][NugetPackagesSeleniumWebdriver400beta02] or later, because it supports Microsoft Edge \(Chromium\).  However, you can control Microsoft Edge \(Chromium\) in all older versions of Selenium, including the current stable Selenium 3 release.  
+[Download the WebDriver language binding of your choice][SeleniumDownloads].  If you are using Selenium, the Microsoft Edge team recommends [Selenium 4.0.0-beta2][NugetPackagesSeleniumWebdriver400beta02] or later, because that version of Selenium supports Microsoft Edge \(Chromium\).  However, you can control Microsoft Edge \(Chromium\) in all older versions of Selenium, including the current stable Selenium 3 release.
```
Comment (bwalderman) [inline microsoft-edge/webdriver-chromium/index.md:75]: I think this link has caused some of the confusion that prompted this bug. It goes to the Selenium download site, so it makes it seem as if the Selenium variants are your only choices as a user. Maybe we should just get rid of this link?

**Review by bwalderman — CHANGES_REQUESTED** [review]

File: microsoft-edge/webdriver-chromium/index.md [hunk 2]
```diff
@@ -18,30 +18,33 @@ WebDriver allows developers to create automated tests that simulate user interac
...
-| Selenium | A popular WebDriver testing framework that test authors use to write end-to-end tests and automate browsers. Selenium can be used on any platform, and is available in Java, Python, C#, Ruby, and JavaScript. |
-| Internet Explorer Driver | An implementation of the WebDriver protocol specifically for Internet Explorer. This product is maintained by the Selenium project. To run legacy end-to-end tests for Internet Explorer, we recommend using Internet Explorer Driver. |
+| Microsoft Edge Driver | Microsoft's implementation of the WebDriver protocol specifically for Microsoft Edge.  Test authors write tests that use WebDriver commands that Microsoft Edge Driver receives.  Microsoft Edge Driver is then responsible for communicating that command to the browser. |
+| A WebDriver testing framework | A framework used by test authors to write end-to-end tests and automate browsers.  Provides a language-specific client driver that translates your code into commands that Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  WebDriver testing frameworks exist for all major platforms and languages.  One such framework is Selenium. |
```
Comment (bwalderman) [inline microsoft-edge/webdriver-chromium/index.md:36]:
```suggestion
| A WebDriver testing framework | A framework used by test authors to write end-to-end tests and automate browsers.  Provides a language-specific interface that translates your code into commands that Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  WebDriver testing frameworks exist for all major platforms and languages.  One such framework is Selenium. |
```

File: microsoft-edge/webdriver-chromium/index.md [hunk 3]
```diff
@@ -59,34 +62,36 @@ To begin automating tests, use the following steps to ensure that the WebDriver
...
-    > For more information about test automation using Microsoft Edge \(EdgeHTML\), navigate to [Microsoft Edge Driver for Microsoft Edge (EdgeHTML)][Webdriver].  
-    -->  
-    
-## Choose a WebDriver language binding  
+## Choose a WebDriver testing framework
+
+After downloading Microsoft Edge Driver, the last component you must download is a WebDriver testing framework.
 
-The last component you must download is a language-specific client driver to translate your code \(Python, Java, C\#, Ruby, JavaScript\) into commands the Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  
+A WebDriver testing framework is used by test authors to write end-to-end tests and automate browsers.  The framework provides a language-specific client driver that translates your code (such as Python, Java, C#, Ruby, or JavaScript) into commands that Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  WebDriver testing frameworks exist for all major platforms and languages.
```
Comment (bwalderman) [inline microsoft-edge/webdriver-chromium/index.md:71]:
```suggestion
A WebDriver testing framework is used by test authors to write end-to-end tests and automate browsers.  The framework provides a language-specific interface that translates your code (such as Python, Java, C#, Ruby, or JavaScript) into commands that Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  WebDriver testing frameworks exist for all major platforms and languages.
```

**Review by bwalderman — APPROVED** [review]
LGTM! Just left a couple suggestions.

File: microsoft-edge/webdriver-chromium/index.md [hunk 4]
```diff
@@ -18,30 +18,33 @@ WebDriver allows developers to create automated tests that simulate user interac
...
-| Selenium | A popular WebDriver testing framework that test authors use to write end-to-end tests and automate browsers. Selenium can be used on any platform, and is available in Java, Python, C#, Ruby, and JavaScript. |
-| Internet Explorer Driver | An implementation of the WebDriver protocol specifically for Internet Explorer. This product is maintained by the Selenium project. To run legacy end-to-end tests for Internet Explorer, we recommend using Internet Explorer Driver. |
+| Microsoft Edge Driver | Microsoft's implementation of the WebDriver protocol specifically for Microsoft Edge.  Test authors write tests that use WebDriver commands that Microsoft Edge Driver receives.  Microsoft Edge Driver is then responsible for communicating that command to the browser. |
+| A WebDriver testing framework | A framework used by test authors to write end-to-end tests and automate browsers.  Provides a language-specific interface that translates your code into commands that Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  WebDriver testing frameworks exist for all major platforms and languages.  One such framework is Selenium. |
```
Comment (Reezaali) [inline microsoft-edge/webdriver-chromium/index.md:36]:
```suggestion
| A WebDriver testing framework | Test authors use a testing framework to write end-to-end tests and automate browsers.  Provides a language-specific interface that translates your code into commands that Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  WebDriver testing frameworks exist for all major platforms and languages.  One such framework is Selenium. |
```

File: microsoft-edge/webdriver-chromium/index.md [hunk 5]
```diff
@@ -59,34 +62,36 @@ To begin automating tests, use the following steps to ensure that the WebDriver
...
     
-    <!--  
-    > [!NOTE] 
-    > For more information about test automation using Microsoft Edge \(EdgeHTML\), navigate to [Microsoft Edge Driver for Microsoft Edge (EdgeHTML)][Webdriver].  
-    -->  
-    
-## Choose a WebDriver language binding  
+## Choose a WebDriver testing framework
+
+After downloading Microsoft Edge Driver, the last component you must download is a WebDriver testing framework.
```
Comment (Reezaali) [inline microsoft-edge/webdriver-chromium/index.md:69]:
```suggestion
After downloading Microsoft Edge Driver, the last component you must download is a WebDriver testing framework. Test authors use WebDriver testing frameworks to write end-to-end tests and automate browsers. The framework provides a language-specific interface that translates your code (such as Python, Java, C#, Ruby, or JavaScript) into commands that Microsoft Edge Driver runs in Microsoft Edge \(Chromium\). WebDriver testing frameworks exist for all major platforms and languages.
```

File: microsoft-edge/webdriver-chromium/index.md [hunk 6]
```diff
@@ -59,34 +62,36 @@ To begin automating tests, use the following steps to ensure that the WebDriver
...
 
-The last component you must download is a language-specific client driver to translate your code \(Python, Java, C\#, Ruby, JavaScript\) into commands the Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  
+A WebDriver testing framework is used by test authors to write end-to-end tests and automate browsers.  The framework provides a language-specific interface that translates your code (such as Python, Java, C#, Ruby, or JavaScript) into commands that Microsoft Edge Driver runs in Microsoft Edge \(Chromium\).  WebDriver testing frameworks exist for all major platforms and languages.
```
Comment (Reezaali) [inline microsoft-edge/webdriver-chromium/index.md:71]:
```suggestion

```

File: microsoft-edge/webdriver-chromium/index.md [hunk 7]
```diff
@@ -368,31 +378,33 @@ docker run -d -p 9515:9515 mcr.microsoft.com/msedge/msedgedriver
 
 For more information, navigate to the [msedgedriver container on Docker Hub][DockerHubMsedgedriver].  
 
+
 ## Testing Internet Explorer
 
-Even though Microsoft Edge supports IE Mode, you can't use Microsoft Edge Driver with Microsoft Edge to test sites in IE Mode.  To test sites that require Internet Explorer, use [Internet Explorer Driver][GithubSeleniumHqWikiIEDriver] with Internet Explorer.
+To test sites that require Internet Explorer, use [Internet Explorer Driver][GithubSeleniumHqWikiIEDriver] with Internet Explorer.  Internet Explorer Driver is maintained by the Selenium project.
+
+Even though Microsoft Edge supports IE Mode, you can't use Microsoft Edge Driver with Microsoft Edge to test sites in IE Mode.
```
Comment (Reezaali) [inline microsoft-edge/webdriver-chromium/index.md:386]:
```suggestion
To test sites that require Internet Explorer, use [Internet Explorer Driver][GithubSeleniumHqWikiIEDriver] with Internet Explorer.  Internet Explorer Driver is maintained by the Selenium project.  Even though Microsoft Edge supports IE Mode, you can't use Microsoft Edge Driver with Microsoft Edge to test sites in IE Mode.
```

**Review by Reezaali — CHANGES_REQUESTED** [review]

**Review by Reezaali — APPROVED** [review]
